### DISCUSSION TITLE
Replace text with user input after template generated

### DISCUSSION THREAD

**Discussion by SammyIsra** [SilentVoid13/Templater#1052] [Help]
Hi all! I have a scenario that I am not sure if is possible using Templater or not.

I have a template called, for example, CatchUpTemplate, which generates a file for me to take notes on a catch up call. These calls are usually between me and one other person, and I'd like to have the name of that person in multiple places in the generated file. How I do this right now is:
1. Generate file from template
2. Go thru every place where I have something like "insert name here", usually 3 or 4 places
3. Replace with name and/or link to the person's profile file that I have

Is it possible for Templater to prompt the user (me) for the name of the person that I am having a catch up call with so that I can generate the file with their name in it already? Bonus points if this person can be selected from the list of file names I have in my specific "People" directory.

**Comment by Zachatoo** [comment]
To get a list of file names in a directory, you can use something like [this](https://zachyoung.dev/snippets/templater#suggester-for-files-in-a-specific-folder).

Here's a more specific example for you:

```js
<%*
const folder = "People";
const peopleFiles = app.vault.getMarkdownFiles().filter(x => x.path.startsWith(folder));
const selectedPerson = (await tp.system.suggester((file) => file.basename, peopleFiles)).basename;
-%>
'I am having call with: [[<% selectedPerson %>]]

Blah blah blah

Remind me to talk with <& selectedPerson %> again at some point.
```
